lib/msal-node/src/config/Configuration.ts

```diff
@@ -84,6 +94,7 @@ export type Configuration = {
     cache?: CacheOptions;
     system?: NodeSystemOptions;
     telemetry?: NodeTelemetryOptions;
+    extensibility?: NodeExtensibilityOptions;
```

Comment:
Same comment here. I'd like to see if there's a way we can expose this only for the Classes for which this feature is supported.

Reply:
Understood. The point I'm trying to make is that this configuration object is used for _all_ flows, including PublicClient flows, and if this is only relevant to one or two flows we should make it an option for only those flows rather than all flows. It's confusing for our customers to provide options that aren't relevant and ultimately get ignored.